tests/integration/utils.go

```diff
@@ -253,7 +255,9 @@ func (p *Process) start() error {
 	go func() {
 		scanner := bufio.NewScanner(pr)
 		for scanner.Scan() {
-			p.t.Logf("[%s %s]: %s", p.name, p.uid, scanner.Text())
```

Comment:
This should already be covered by using Expect (see config_test.go)

Reply:
Hey @sgotti I'm struggling to figure out how to use the expect interface to try matching against what I need here.

I need to verify that this string occurs after a particular place in the code (so flush the buffer, then run the assertion) and I need to perform multiple assertions for several strings that may appear out of order.

Can you help me figure out how to do this with Expect?

Reply:
Figured this out actually, thanks!